test/form/samples/treeshake-unary/_expected.js

```diff
@@ -0,0 +1,29 @@
+console.log(true);
+console.log(false);
+console.log(true);
```

Comment:
If we go for the smallest possible representation, we could do what minifiers do and write `true` as `!0` and `false` as `!1`. Considering some of the other transformations we do, I think this should not be an issue.

Reply:
Yeah, that makes sense!